lib/puppet/http/service/compiler.rb

```diff
@@ -102,6 +102,8 @@ def get_status(name)
       },
     )
 
+    @session.process_response(response)
```

Comment:
This might be in the wrong commit? Also, the commit that this is in needs a commit message.

Reply:
Updated

Reply:
Need to update commit summary